leveler/commands/other.py

```diff
@@ -32,12 +32,20 @@ async def rep(self, ctx, *, user: discord.Member = None):
         if user and user.bot:
             await ctx.send("**You can't give a rep to a bot!**")
             return
+        antirepfarm = await self.config.guild(ctx.guild).antirepfarm()
+        if antirepfarm == True and user:
+            if "lastrep" in org_userinfo:
+                if user.id == org_userinfo["lastrep"]:
+                    return await ctx.send("**You already gave a rep point to this user!**")
```

Comment:
That's a nice pyramid you have here.
```suggestion
        if user and await self.config.guild(ctx.guild).antirepfarm() and user.id == org_userinfo.get("lastrep"):
            await ctx.send("**You already gave a rep point to this user!**")
            return
```